apps/wing-console/console/server/src/router/app.ts

```diff
@@ -601,12 +601,21 @@ export const createAppRouter = () => {
         }),
       )
       .mutation(async ({ ctx, input }) => {
-        ctx.emitter.emit("openFileInEditor", {
-          path: input.path,
-          line: input.line,
-          column: input.column,
-        });
+        if (ctx.emitter.listenerCount("openFileInEditor") > 0) {
+          ctx.emitter.emit("openFileInEditor", {
+            path: input.path,
+            line: input.line,
+            column: input.column,
+          });
+          return;
+        }
+        const launch = require("launch-editor");
```

Comment:
Can we use `const launch = await import("launch-editor");` instead? Requires aren't ESM compatible, and would like to stay future proof.

Reply:
fixed!